CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+Please see [doc/docs/project/CHANGELOG.md
```

Comment:
Well there used to be a man/ directory in the repo too (it has since disappeared though), and to me things like man pages are also documentation (and I don't know if you'd ever want to include haddocks as well), but I don't think those should necessarily be mixed in with anything you maybe potentially host on readthedocs,org. Man pages and haddocks just feel like a different category of documentation to me, but they are still documentation of course. At least if all categories of documentation are under doc/ (even with subdirectories), rather than scattered about, in my mind it made it easier to find any possible category of docs without having to crawl the directories too much.